Tasks/JenkinsDownloadArtifacts/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -51,5 +64,53 @@
+  "loc.messages.armUrlCannotBeEmpty": "arm Url must be a non empty string.",
+  "loc.messages.authorityUrlCannotBeEmpty": "authority must be a non empty string.",
+  "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for azure. Status code: %s, status message: %s",
+  "loc.messages.CallbackCannotBeNull": "callback cannot be null.",
+  "loc.messages.VMNameCannotBeNull": "vmName cannot be null or undefined and it must be of type string.",
+  "loc.messages.InvalidValue": "%s is not a valid value. The valid values are: %s",
+  "loc.messages.VmExtensionNameCannotBeNull": "vmExtensionName cannot be null or undefined and it must be of type string.",
+  "loc.messages.ExpandShouldBeOfTypeString": "expand must be of type string.",
+  "loc.messages.ExtensionParametersCannotBeNull": "extensionParameters cannot be null or undefined.",
+  "loc.messages.LoadBalancerNameCannotBeNull": "'loadBalancerName cannot be null or undefined and it must be of type string.'",
+  "loc.messages.ParametersCannotBeNull": "parameters cannot be null or undefined.",
+  "loc.messages.NetworkInterfaceNameCannotBeNull": "networkInterfaceName cannot be null or undefined and it must be of type string.",
+  "loc.messages.NetworkSecurityGroupNameCannotBeNull": "networkSecurityGroupName cannot be null or undefined and it must be of type string.",
+  "loc.messages.SecurityRuleNameCannotBeNull": "securityRuleName cannot be null or undefined and it must be of type string.",
+  "loc.messages.SecurityRuleParametersCannotBeNull": "securityRuleParameters cannot be null or undefined.",
+  "loc.messages.DeploymentNameCannotBeNull": "deploymentName cannot be null or undefined and it must be of type string.",
+  "loc.messages.CredentialsCannotBeNull": "'credentials' cannot be null.",
+  "loc.messages.SubscriptionIdCannotBeNull": "'subscriptionId' cannot be null.",
+  "loc.messages.InvalidResponseLongRunningOperation": "Invalid response received for fetching status of a long running operation.",
+  "loc.messages.TimeoutWhileWaiting": "Timed out while waiting",
+  "loc.messages.ResourceGroupCannotBeNull": "resourceGroupName cannot be null or undefined and it must be of type string.",
+  "loc.messages.ResourceGroupExceededLength": "\"resourceGroupName\" should satisfy the constraint - \"MaxLength\": 90",
+  "loc.messages.ResourceGroupDeceededLength": "\"resourceGroupName\" should satisfy the constraint - \"MinLength\": 1",
+  "loc.messages.ResourceGroupDoesntMatchPattern": "\"resourceGroupName\" should satisfy the constraint - \"Pattern\": /^[-\\w\\._\\(\\)]+$/",
+  "loc.messages.VMSSNameCannotBeNull": "VMSS name cannot be null or undefined and it must be of type string.",
+  "loc.messages.GetVMSSFailed": "Failed to get VMSS details with resource group %s and name %s. Error: %s",
+  "loc.messages.VMSSDoesNotHaveCustomImage": "VMSS %s can not be updated as it uses a platform image. Only a VMSS which is currently using a custom image can be updated.",
```

Comment:
Are these required?

Reply:
no i'll remove these and all other belonging to vmss